plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/frontend.ts

```diff
@@ -281,6 +288,28 @@ const addToCartWithOptionsStore = store(
 					childProductId
 				);
 			},
+			handleQuantityChange: (
+				event: HTMLElementEvent< HTMLInputElement >
+			) => {
+				const inputData = getInputData( event );
+				if ( ! inputData ) {
+					return;
+				}
+				const { childProductId, maxValue, minValue, currentValue } =
+					inputData;
+
+				const newValue = Math.min(
+					maxValue ?? Infinity,
+					Math.max( minValue, currentValue )
+				);
+
+				addToCartWithOptionsStore.actions.setQuantity(
```

Comment:
@Aljullu I think this the too much recursion we are [discussing in Slack](https://woocommercecommunity.slack.com/archives/C03MCL9RTAR/p1751984653541299)